### PR TITLE
Feat: scheduled change request dialogs

### PR DESCRIPTION
**Pull request by andreas-unleash**
Creates the Apply and Reject scheduled change request dialogs

Closes # [1-1584](https://linear.app/unleash/issue/1-1584/add-modal-for-apply-now)

Closes # [1-1586](https://linear.app/unleash/issue/1-1586/reject-changes-dialog)

Screenshot 2023-11-03 at 14 43 17
Screenshot 2023-11-03 at 14 43 28

UI e2e tests will be in a follow up PR

### CODE REVIEW

File: frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            secondaryButtonText='Cancel'
+            open={open}
+            onClose={onClose}
+            onClick={() => onConfirm()}
+            fullWidth
+        >
+            <StyledAlert icon={false}>
+                There is a scheduled time to apply these changes set for{' '}
+                <strong>
+                    <br />
```
Comment (thomasheartman) [inline frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx:52]: Why are we using the `br` element? If we need to space this out, we should do that with styling, not html, I think.

**Review by thomasheartman — APPROVED** [review]
Looks pretty good! I'd prefer to avoid using HTML for styling (so don't use the `br` element), but otherwise good. I've already voiced my concerns for the time handling, but we don't need to address that just yet.

File: frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx (hunk 1 above)
Comment (andreas-unleash) [reply frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx:52]: The reason I user `br` is that the designs look to me to require it be in a new line.  What would be a better way to do it?

Comment (thomasheartman) [reply frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestScheduledDialogs/ChangeRequestScheduledDialogue.tsx:52]: Yeah, this sounds like a styling thing to me. The way I've always learnt is that visual-only things should be done with CSS, not HTML. So for instance, if you need the `strong` element to be on its own line, you could use css to give it `display: block` instead. It's not a super important thing, but more of a ... best practices thing. At least as far as I'm aware.

**Pull request merged**